src/helpers/api.js

```diff
@@ -34,34 +34,29 @@ if (process.env.NODE_ENV !== 'production') {
  * Pushes an array of messages to the notification handler.
  *
  * @param {Array} messages - An array of messages.
- * @param {Object} options - Display options for the notifications.
  */
-function showNotifications(messages, options) {
+function showNotifications(messages) {
```

Comment:
Please adjust the wording to use `notifications` as `message` is one part of one notification instance